package/kernel/mt-drivers/mt_wifi/files/mt7603.dat

```diff
@@ -212,7 +212,7 @@ MACRepeaterOuiMode=2
 MUTxRxEnable=0
 NoForwarding=0
 NoForwardingBTNBSSID=0
-own_ip_addr=192.168.1.1
+own_ip_addr=10.10.10.254
```

Comment:
I don't use this package, so I'm not sure if these changes are correct.
Default bridge name on OpenWrt is called `br-lan`, and default IP is `192.168.1.1`, so any reason changing these params?

Reply:
In my memory, `br0` is the name used in very old openwrt releases.
However, this option is only used for radius, but radius can not be built after time API changed in 5.x kernel.

Reply:
I'm not sure either. These commits are basically dirty checkouts with minor cleanup, which is dirty af... I'll test if the changes mentioned are redundant.